cmake/shared/CodeAnalysis.cmake

```diff
@@ -0,0 +1,24 @@
+add_custom_target(analyse
```

Comment:
De we need to `EXCLUDE_FROM_ALL` these targets so they're not run on a plain `make`?

Reply:
https://cmake.org/cmake/help/v3.15/command/add_custom_target.html

For custom targets you add to all, not exclude.

Reply:
Ah, you're right. I was running "Build Solution" on Visual Studio instead of building `ALL_BUILD`